containers/kubernetes/how-to/monitor-cluster.mdx

```diff
@@ -18,27 +18,57 @@ categories:
-    * Cluster information
-    * Auto Upgrade
-    * Cluster tags
-    * Renewal of the kubeconfig file to restore access permissions
-    * Download of the kubeconfig file
-    * The option to delete the cluster
+[Cockpit](/observability/cockpit/quickstart/) empowers you to monitor the clusters' control plane, nodes, managed resources and the cluster system applications.
+* Detect incidents and diagnose failures in the earliest stages
+* Get new insights about control plane and worker nodes usage such as CPU, memory, to optimize your resource allocation
+* Achieve a better understanding of your infrastructure by analyzing data on your API server, system applications, up and down scaling behavior
 
-If you click the **Dashboard** button, you are taken to your cluster's Kubernetes dashboard. Here you can troubleshoot your containerized application, manage the cluster resources, get an overview of applications running on your cluster, and create or modify certain resources.
+## How to access your Cockpit Grafana dashboard
 
-If you click the **Pools** tab, you are able to add, edit or delete a pool on your cluster.
+<Message type="important">
+  You must have [activated](/observability/cockpit/how-to/activate-cockpit/) your Cockpit. Cockpit’s free plan retention time being 31 days for metrics and 7 days for logs.
+</Message>
+
+1. Open your Cockpit in the [Scaleway console](https://console.scaleway.com/cockpit/overview).
+2. Click **Manage Scaleway data**. The Grafana interface opens in a new browser tab.
+3. Log into Grafana using your Cockpits' Grafana user and password. The list of your Cockpit dashboards displays.
+4. Click **Kubernetes Cluster Overview** in the list of dashboards. A status overview of your cluster displays. This dashboard allows you to monitor multiple components of your Kubernetes cluster: control-plane, nodes, managed resources and cluster system applications.
+    <Lightbox src="scaleway-k8s-cluster-metrics.webp" alt="" />
+    <Message type="tip">
+      Select the cluster you want to display from the drop-down list at the top of the page to change the currently displayed cluster.
+    </Message>
+
+### Monitoring of the kube-apiserver
+
+The cluster overview dashboard offers real-time monitoring capabilities for the `kube-apiserver` within your Kubernetes cluster. Serving as a crucial component on the control plane, the `kube-apiserver` acts as the gateway to the Kubernetes API, allowing you to interact with the cluster.
+
+In cases where your clusters are large and containing numerous resources such as nodes, pods, and custom resources (CRDs), or when experiencing a large number of simultaneous requests to the apiserver from controllers like argocd and velero,  it is possible for the CPU and memory usage to spike. Consequently, the performance of the apiserver may noticeably decline, leading to sluggishness or unresponsiveness. You might encounter errors like `"EOF"` when using `kubectl`.
```

Comment:
This sentence is a bit long, maybe try making it shorter?

Reply:
Can't make it shorter. If you have any idea how, let me know

Reply:
The CPU and memory usage might spike for large clusters containing many resources (nodes, pods, and custom resources) or experiencing many simultaneous requests to the API server from controllers such as argocd and velero. 

Something like that maybe ?